admin_guide/topics/deploying-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,262 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='deploying-cluster-auto-scaler-{context}']
+= Deploying the auto-scaler components on your cluster
+
+After you create the Launch Configuration and Auto Scaling group, you can deploy
+the auto-scaler components onto the cluster.
+
+.Prerequisites
+
+* Install a {product-title} cluster in AWS.
+* Create a primed image.
+* Create a Launch Configuration and Auto Scaling group that reference the primed
+image.
+* Add the following parameter to the inventory file that you used to create the
+cluster, by default *_/etc/ansible/hosts_*:
++
+----
+openshift_master_bootstrap_auto_approve=true
```

Comment:
We need to mention that you may need to re-run the AWS playbooks on the existing cluster to deploy the bootstrap auto approver components if this component is `False`.

Reply:
The more I think about this is... as listed as prerequisite it should say, your existing cluster needs the bootstrap autoapprover components to be running. You can check this with:

```console
$ [ec2-user@ip-172-18-9-89 ~]$ oc get pods --all-namespaces | grep bootstrap-autoapprover
openshift-infra         bootstrap-autoapprover-0                         1/1       Running   0          5h
```

Reply:
That's fair, but I think we need to move it out of the prereqs and into the steps.